### DISCUSSION TITLE
Unable to install Statamic on Digital Ocean Ubuntu droplet

### DISCUSSION THREAD

**Discussion by vhbelvadi** [statamic/cms#7731] [Q&A] [answered]
I created a digital ocean droplet with Ubuntu and followed the instructions [on this page](https://statamic.dev/installing/ubuntu) until installing Composer which went alright when I ran the `composer` command. The next step is to `cd /var/www` but this gives me an error saying the directory doesn’t exist. I’m in root and when I `ls` I see a folder called ‘snap’.

**Comment by joshuablum** [comment, marked as answer]
Hey @vhbelvadi,

You could create that directory of course (`mkdir /var/www`). But that alone will probably not solve your problem.

Make sure that your server is **properly set up and configured with a web server** (like nginx).

Did you create and configure your droplet manually? We recommend to use a service like [Laravel Forge](https://forge.laravel.com) or [Ploi.io](https://ploi.io)(they even offer a free tier) to provision and manage the server. All necessary packages and dependencies to set up by those services.

Otherwise I would suggest to follow a guide like [this](https://www.digitalocean.com/community/tutorials/how-to-install-linux-nginx-mysql-php-lemp-stack-on-ubuntu-22-04) to set up your droplet.

Hope that helps :)

**Reply by vhbelvadi** [reply]
Thanks. It seems to be working alright after I redid the Nginx setup.
